.docker/scholarly-articles/data/article1/1.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<item xmlns="http://libero.pub" xmlns:jats="http://jats.nlm.nih.gov" xmlns:xlink="http://www.w3.org/1999/xlink">
+<item xmlns="http://libero.pub" xmlns:jats="http://jats.nlm.nih.gov" xmlns:xlink="http://www.w3.org/1999/xlink"
+    xml:base="https://prod-elife-published.s3.amazonaws.com/articles/09560/">
```

Comment:
the bucket rather than the CDN; which is good to avoid (most of) the caching layers

Reply:
also more realistic in case the `content-store` will download this